test/infrastructure/docker/internal/controllers/dockermachine_controller.go

```diff
@@ -307,45 +329,27 @@ func (r *DockerMachineReconciler) reconcileNormal(ctx context.Context, cluster *
 		// but bootstrapped is never set on the object. We only try to bootstrap if the machine
 		// is not already bootstrapped.
 		if err := externalMachine.CheckForBootstrapSuccess(timeoutCtx, false); err != nil {
-			bootstrapData, format, err := r.getBootstrapData(timeoutCtx, machine)
-			if err != nil {
-				return ctrl.Result{}, err
-			}
-
-			// Setup a go routing to check for the machine being deleted while running bootstrap as a
-			// synchronous process, e.g. due to remediation. The routine stops when timeoutCtx is Done
-			// (either because canceled intentionally due to machine deletion or canceled by the defer cancel()
-			// call when exiting from this func).
-			go func() {
```

Comment:
Why are we dropping this code? this change seems unrelated to this PR and it drops something that was introduced to get rid of some flakes/issue with deletion

Reply:
Likely a rebase error, I can go ahead and add that back.